dags/analyzer_helper/tests/unit/test_transform_raw_info.py

```diff
@@ -0,0 +1,370 @@
+        raw_data = {
+            "post_id": 6261,
+            "author_id": 6168,
+            "created_at": "2023-09-11T21:42:43.553Z",
+            "category_id": 500,
+            "topic_id": 6134,
+            "reactions": [6263],
+            "replied_post_id": None
+        }
+        result = self.transformer.create_data_entry(raw_data, interaction_type="reaction", interaction_user=6263)
+        print("test_create_data_entry_with_reaction \n", result)
+        self.assertEqual(result['author_id'], "6263")
+        self.assertEqual(result['interactions'][0]['name'], "reaction")
+        self.assertEqual(result['interactions'][0]['type'], 'emitter')
+        self.assertEqual(result['interactions'][0]['users_engaged_id'], [str(raw_data['post_id'])])
+        self.assertIsInstance(result['date'], datetime.datetime)
+        self.assertEqual(result['source_id'], str(raw_data['post_id']))
+        self.assertEqual(result['metadata']['category_id'], raw_data['category_id'])
+        self.assertEqual(result['metadata']['topic_id'], raw_data['topic_id'])
+        self.assertEqual(len(result['actions']), 0)
+
+    def test_transform_data_with_replied_user(self):
+        raw_data = [
+                {
+                    "post_id": 6262,
+                    "author_id": 6168,
+                    "created_at": "2023-09-11T21:41:43.553Z",
+                    "author_name": "Test Author Name1",
+                    "reactions": [],
+                    "replied_post_id": 6512,
+                    "replied_post_user_id": 4444,
+                    "topic_id": 6134,
+                }
+        ]
+
+        expected_result = [
+            {
+                'author_id': '6168',
+                'date': datetime.datetime(2023, 9, 11, 21, 41, 43, 553000, tzinfo=datetime.timezone.utc),
+                'source_id': '6262',
+                'metadata': {
+                    'category_id': None,
+                    'topic_id': 6134,
+                    'bot_activity': False
+                },
+                'actions': [
+                    {
+                        'name': 'message',
+                        'type': 'emitter',
+                    }
+                ],
+                'interactions': [
+                    {
+                        'name': 'reply',
+                        'type': 'emitter',
+                        'users_engaged_id': ['4444'],
+                    }
+                ]
+            },
+            {
+                'author_id': '4444',
+                'date': datetime.datetime(2023, 9, 11, 21, 41, 43, 553000, tzinfo=datetime.timezone.utc),
+                'source_id': '6262',
+                'metadata': {
+                    'category_id': None,
+                    'topic_id': 6134,
+                    'bot_activity': False,
+                },
+                'actions': [],
+                'interactions': [
+                    {
+                        'name': 'reply',
```

Comment:
The reply contains always a message as a reply so the `actions` field should be filled too.

Reply:
Agree. It means that we missed it in the case of discord as well? 

https://github.com/TogetherCrew/airflow-dags/blob/fd49978aa010ecec558f5adb25de1b17d76ee9a3/dags/analyzer_helper/tests/integration/test_discord_transform_raw_data.py#L97